documentation/src/main/asciidoc/user_guide/locking.adoc

```diff
@@ -29,112 +36,101 @@ NOTE: If the operation is conditional and it fails on the primary owner, it is n
-
-
-. Thread1: tx.begin()
-. Thread1: cache.get(k) returns v
-. Thread2: tx.begin()
-. Thread2: cache.get(k) returns v
-. Thread2: cache.put(k, v2)
-. Thread2: tx.commit()
-. Thread1: cache.get(k) returns v2!
 
-With REPEATABLE_READ, step 7 will still return v. So, if you're going to retrieve the same key multiple times within a transaction, you should use REPEATABLE_READ.
+Isolation level affects what transactions can read when running concurrently with other transaction.
+Check section <<tx:isolation-levels>> for more details about it.
 
 ==== The LockManager
-The LockManager is a component that is responsible for locking an entry for writing.
-The LockManager makes use of a LockContainer to locate/hold/create locks.
-LockContainers come in two broad flavours, with support for lock striping and with support for one lock per entry.
+The `LockManager` is a component that is responsible for locking an entry for writing.
+The `LockManager` makes use of a `LockContainer` to locate/hold/create locks.
+`LockContainers` come in two broad flavours, with support for lock striping and with support for one lock per entry.
 
 ==== Lock striping
-Lock striping entails the use of a fixed-size, shared collection of locks for the entire cache, with locks being allocated to entries based on the entry's key's hash code.  Similar to the way the JDK's ConcurrentHashMap allocates locks, this allows for a highly scalable, fixed-overhead locking mechanism in exchange for potentially unrelated entries being blocked by the same lock.
+Lock striping entails the use of a fixed-size, shared collection of locks for the entire cache, with locks being allocated to entries based on the entry's key's hash code.
+Similar to the way the JDK's `ConcurrentHashMap` allocates locks, this allows for a highly scalable, fixed-overhead locking mechanism in exchange for potentially unrelated entries being blocked by the same lock.
 
-The alternative is to disable lock striping - which would mean a _new_ lock is created per entry.  This approach _may_ give you greater concurrent throughput, but it will be at the cost of additional memory usage, garbage collection churn, etc. 
+The alternative is to disable lock striping - which would mean a _new_ lock is created per entry.
+This approach _may_ give you greater concurrent throughput, but it will be at the cost of additional memory usage, garbage collection churn, etc.
 
 .Default lock striping settings
-IMPORTANT: From Infinispan 5.0, lock striping is disabled by default, due to potential deadlocks that can happen if locks for different keys end up in the same lock stripe. Previously, in Infinispan 4.x lock striping used to be enabled by default.
+IMPORTANT: From Infinispan 5.0, lock striping is disabled by default, due to potential deadlocks that can happen if locks for different keys end up in the same lock stripe.
```

Comment:
I'll remove the sentence. 
The deadlock can happen in non-tx cache but it is not caused by the lock striping.